benchmark.py

```diff
@@ -183,12 +182,15 @@ def repo(repository, branch=None):
         os.chdir(prev_dir)
 
 
-def benchmark(project_info, force=False):
+def benchmark(project_info, force=False, keep_env=False):
     current_commits = {}
     update_triggered_by = []
 
     db = BenchmarkDatabase(project_info["name"])
-
+    
+    #remove previous repos and clone fresh ones to avoid trouble.
```

Comment:
so here is where you probably want to figure out the repo_dirs name based on the project name, than pass that name to remove_repos_dir and the subsequenct `with repos`